boards/arm/bl654_usb/Kconfig

```diff
@@ -0,0 +1,18 @@
+# BL654 USB adapter board configuration
+
+# Copyright (c) 2021 Laird Connectivity
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_ENABLE_DCDC
+	bool "Enable DCDC mode"
+	select SOC_DCDC_NRF52X
+	default y
+	depends on BOARD_BL654_USB
+
+config BOARD_HAS_NRF5_BOOTLOADER
+	bool "Board has nRF5 bootloader"
+	default y
+	help
+	  If selected, applications are linked so that they can be loaded by Nordic
+	  nRF5 bootloader.
+	depends on BOARD_BL654_USB
```

Comment:
Would remove this and always set the load offset to 0x1000, the MBR cannot be removed from this device so switching this off is an invalid build configuration

Reply:
Updated.